examples/classic-typescript/docs/intro.md

```diff
@@ -14,7 +14,7 @@ Or **try Docusaurus immediately** with **[docusaurus.new](https://docusaurus.new
 
 ### What you'll need
 
-- [Node.js](https://nodejs.org/en/download/) version 16.14 or above:
+- [Node.js](https://nodejs.org/en/download/) version 18.0 or above:
```

Comment:
Examples are generated with a script and are used to populate stackblitz/codesandbox templates. 

They should represent the stable version of Docusaurus and shouldn't be updated manually.

These example changes should be reverted because examples are still using Docusaurus v2 (which supports Node 16.14)

They are basically new sites initialized with `npx create-docusaurus@latest`, so if you modify `create-docusaurus` these will be self-updated on next regeneration after a stable release. We don't have examples for unstable/next/alpha/beta/rc/canary versions.